### DISCUSSION TITLE
Wettervorhersage Regen & Schnee

### DISCUSSION THREAD

**Discussion by OE3MNS** [Daveiano/weewx-wdc#231] [Q&A]
Hallo, 

ist es möglich (Bitte) bei der Wettervorhersage eine extra Zeile für Schnee (in cm) zu erstellen ? Es ist für meine Webseiten Besucher verwirrend wenn bei Regen über 100mm steht. 

![IMG_0471](https://github.com/Daveiano/weewx-wdc/assets/147713099/a954d452-ce2a-4b2e-a022-cf2ae9a7b297)

Herzlichen Dank 
Lg Michi

**Comment by Daveiano** [comment]
Hi Michi,

ja das stimmt, ein wenig verwirrend.

Welchen Anbieter (weatherundeground, OWM, Aeris, etc) nutzt du? Teilweise sind die Informationen die geliefert werden, unterschiedlich. 
Ich schau mal, was sich machen lässt!

**Reply by OE3MNS** [reply]
Aeris, vielen Dank.
LG

**Comment by Daveiano** [comment]
Hi Michi,

schau mal wie findest du das:

![GH-231](https://github.com/Daveiano/weewx-wdc/assets/3184897/6649595d-7625-4f46-9071-6e2b95f65a05)

Ich habe zuallerst die Zeilenbeschriftung von "Regen" zu "Niederschlag" geändert (Regen war hier eigentlich falsch da dort die Menge allen Niederschlages gezeigt wird). Und außerdem habe ich jeweils eine extra Zeile für die jeweilisge Menge Regen/Schnee eingefügt (wie auch in dem Bsp. Skin der mit der forecast Erweiterung mitgeliefert wird).

**Reply by OE3MNS** [reply]
Perfekt! Vielen Dank.

**Reply by Daveiano** [reply]
Super! Dann kommt das so in nächste Release!
